docs/libertyRun.md

```diff
@@ -0,0 +1,38 @@
+## libertyRun task  
+Start a Liberty server in foreground. The server instance will be automatically created if it does not exist.  
+**NOTE!** To view shutdown messages when terminating `libertyRun` with a Ctrl-C, run `libertyRun` like this:  
+```
+gradle libertyRun --no-daemon
+```
+  
+### 0% Executing?
+While running this task, Gradle will show:  
+```
+<-------------> 0% EXECUTING  
+> :libertyRun
+```  
+This is expected behavior because the task will neither progress nor stop as long as the server/process is running. The "build" will successfully finish with an external `libertyStop` or a Ctrl-C break (if ran with `--no-daemon`).
+
+### What is the Gradle Daemon and why --no-daemon?
+The Gradle Daemon is a long-running background process designed to help speed up the build process. It does so by avoiding constant JVM startup costs while caching information about the project. This behavior is default until specified otherwise.  
+If running with the daemon (default), users who like to Ctrl-C their `libertyRun` task will find that this kills their daemon, ends `libertyRun` prematurely, and most importantly, leaves the server still running. In this state, it's best to restart your server. You will have to execute a separate `libertyStop` command to stop the server before starting it again.  
```

Comment:
we need to fix up this sentence, since we are stopping the server now.   How about something along the lines of:
If you are running with the daemon (default), if you stop the `libertyRun` task with a CTRL-C, the daemon stops and you will immediately loose console output. The server will be stopped, but this might take a few seconds longer and you will not receive any feedback on the Gradle console.